ui/component/common/card.jsx

```diff
@@ -46,18 +50,22 @@ export default function Card(props: Props) {
               {subtitle && <div className="card__subtitle">{subtitle}</div>}
             </div>
           </div>
-          {expandable && (
-            <div className="section--padded">
-              <Button
-                button={'alt'}
-                aria-label={__('More')}
-                icon={expanded ? toCapitalCase(ICONS.SUBTRACT) : toCapitalCase(ICONS.ADD)}
-                onClick={() => setExpanded(!expanded)}
-              />
-            </div>
-          )}
+          <div>
+            {titleActions && <div className="section--padded">{titleActions}</div>}
```

Comment:
I'm not that big of a fan of `titleActions`, but I think it's fine to keep. I don't think `headerActions` should be added though.

Reply:
not section--padded though, just `.card__header-actions` (or whatever) (even if duplicating CSS)